sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseNewActivity.java

```diff
@@ -80,8 +81,16 @@ public static void startActivityWithEmptyReportDate(Context fromActivity) {
 		BaseEditActivity.startActivity(fromActivity, CaseNewActivity.class, buildBundleWithEmptyReportDate());
 	}
 
-	public static void startActivityFromContact(Context fromActivity, String contactUuid) {
-		BaseEditActivity.startActivity(fromActivity, CaseNewActivity.class, buildBundleWithContact(contactUuid));
+	public static void startActivityFromContact(Context fromActivity, Contact contact) {
+		if (contact.getContactClassification() == ContactClassification.UNCONFIRMED) {
+			NotificationHelper.showNotification(
+				getActiveActivity(),
+				NotificationType.WARNING,
+				getActiveActivity().getResources().getString(R.string.message_contact_to_case_confirmation_required));
```

Comment:
Please use the same message that we use in the web app with I18nProperties; no need to duplicate it in the mobile app